lib/Dialect/FIRRTL/Transforms/ModuleInliner.cpp

```diff
@@ -187,7 +211,11 @@ class MutableNLA {
         llvm::dbgs() << llvm::formatv("{0:x-}", a);
```

Comment:
not patch related: I don't know that dump functions (which should be named dump not dumpState for consistency) need to wrap their bodies in LLVM_DEBUG.  It would be worth checking upstream to see the accepted behavior.

Reply:
Checked. It's not. I'll land a fix for this after I get this PR in. Good suggestion.

Reply:
The convention is to use `llvm::errs()` and not do any wrapping with `LLVM_DEBUG`. I was trying to use `llvm::dbgs()` thinking that was better than `llvm::errs()` hence requiring the macro to wrap it. Fixed in cbb5767d0ac284cfec77386ffa9c712edea554c9.